README.md

```diff
@@ -108,6 +109,20 @@ The HOT Export pipeline depends on a number of third-party tools.
 
 <code>$ sudo apt-get install default-jre zip unzip</code>
 
+Install Qt 5.5:
+
+<code>$ sudo add-apt-repository ppa:beineri/opt-qt55-trusty </code>
+
+<code>$ sudo apt-get update </code>
+
+<code>$ sudo apt-get install qt55base </code>
+
+<code>$ source /opt/qt55/bin/qt55-env.sh </code>
```

Comment:
It's not clear why is Qt5.5 needed, and why are you adding something to the environment with `source` as it will be lost when you close theterminal session